compiler/src/iree/compiler/DispatchCreation/SinkReshapes.cpp

```diff
@@ -50,7 +50,8 @@ struct SinkReshapesPass final
 /// we just approximate it (and try to be optimistic)
 static bool isFusableUsingTileAndFuse(Operation *producer,
                                       Operation *consumer) {
-  return llvm::isa_and_nonnull<IREE::LinalgExt::LinalgFusionOpInterface,
+  return IREE::LinalgExt::isBitTruncateOp(producer) ||
```

Comment:
Its confusing here cause the truncate ops are fusible using tile and fuse, so this shouldnt be needed. What was this for (there is every chance that I wrote this to start with). If you know might be worth adding a lit test.